Makefile

```diff
@@ -155,36 +156,51 @@ dist-ui: build-ui
 
 release: dist-ui build-server ## Build wheel file for release
 
-container-image: ## Build Elyra stand-alone container image
+elyra-image:
+	## Build Elyra stand-alone container image
```

Comment:
Not sure if this was intentional, but moving the double-hash comment from the target line to the line below, winds up removing the target (and its comment) from the help text produced when just typing `make`.  

This same comment applies to all of the xxx-image and publish-xxx-image targets.

Reply:
Similar to what we do in EG, I believe most of the folks will build/publish the set of container images, and contributors might build individual ones and will be familiar with the internal targets